src/Abstractions/Activities/ActivityExtensions.cs

```diff
@@ -104,6 +104,22 @@ public static class ActivityExtensions
 		public static Activity SetMetadata(this Activity activity, string metadata) =>
 			activity.SetTag(ActivityTagKeys.Metadata, metadata);
 
+		/// <summary>
+		/// Set's the activities parent as the dependent property.
+		/// </summary>
+		/// <returns>This property will only be set if the parent exists.</returns>
+		public static Activity SetParentAsDependent(this Activity activity)
+		{
+			Activity? parent = activity.Parent;
```

Comment:
With OTel you already have all links between activities, there's no need for an extra dimension.

Reply:
What existing dimension captures the name of the parent activity?

Reply:
In logs you have the parent span ID. Adding freeform string dimension to a metric may result in cardinality explosion, we shouldn't do that. Do you have a particular scenario where you want this?

Reply:
This is for Metrics, not for Logs or Traces.

Essentially, I want to be able to automatically find (in dashboards), all the activities (by name) that a particular activity depends on. In this context, "depends on" will be it's child activities.

i.e. Let's say I have a parameterized dashboard that takes an activity name. I put in the name "MyEndpoint" and it shows that activity and also the activities that "MyEndpoint" depends on, by setting the Dependent to "MyEndpoint".

As you've mentioned we can't use anything unique to a particular execution of an activity, such as span, because of the cardinality. Furthermore, it's just wouldn't work for this scenario - we need something unaffected by aggregation.

Reply:
We discussed offline: there is no way currently in OTEL to get this information for metrics.